Cannot compile GridLAB-D with HELICS support

Initial Checklist prior to submitting the issue. Check the bellow boxes to the best of your knowledge:

- [x] I have not found an open issue describing the same problem.
- [x] I went through "CONTRIBUTING.MD" before deciding to submit the issue.
- [x] While building GridLAB-D, I followed all steps as detailed in GridLAB-D Wiki.

------------------------------------------------------------------------------------------------------------

## Please describe what was the cause of the issue?

I'm trying to compile GridLAB-D with HELICS support. I'm compiling code from the `develop` branch, which includes support for the `--with-helics` configure option. However, when configuring the code for compilation, I get the `checking for HELICS... no` message even though HELICS has been compiled and installed locally. I'm setting `--with-helics=/usr/local`, which is where HELICS is installed.

## Please provide the steps to reproduce.

1. Configure, compile, and install HELICS with the `BUILD_CXX_SHARED_LIB=ON` and `CMAKE_INSTALL_PREFIX=/usr/local` cmake options set.
1. Configure, compile, and install GridLAB-D from the `develop` branch with the `--with-helics=/usr/local` configure option set.

## Please provide the exact error/warnining/failure message printed on the console.

When configuring the source code for compilation, I get the `checking for HELICS... no` message included in the output. Also, when I attempt to run the GridLAB-D portion of the [first HELICS tutorial](https://github.com/GMLC-TDC/HELICS-Tutorial/tree/master/tutorials/1-DistributionFederation-ManualStart) I get the following error.

```
DistributionSim_B2_G_1.glm(62): class 'helics_msg' is not known
DistributionSim_B2_G_1.glm(62): load failed at or near 'object helic...'
FATAL    [INIT] : shutdown after command line rejected
```

Thanks @afisher1 for pointing me in the direction of `config.log`. Any idea what these errors are?

```
configure:11161: checking for HELICS
configure:11178: g++ -o conftest -g -O0 -w -std=c++11   -I/usr/local/include -g -O0 -w  -L/usr/local/lib conftest.cpp   -lhelics-shared -lzmq -lboost_chrono -lboost_date_time -lboost_filesystem -lboost_program_options -lboost_system -lboost_timer -lboost_unit_test_framework >&5
In file included from /usr/local/include/helics/core/helics-time.hpp:8:0,
                 from /usr/local/include/helics/core/core-data.hpp:8,
                 from /usr/local/include/helics/application_api/helicsTypes.hpp:14,
                 from conftest.cpp:104:
/usr/local/include/helics/common/timeRepresentation.hpp:134:36: error: missing terminating ' character
                                  10'000,
                                    ^~~~~
/usr/local/include/helics/common/timeRepresentation.hpp:135:37: error: missing terminating ' character
                                  100'000,
                                     ^~~~~
/usr/local/include/helics/common/timeRepresentation.hpp:139:43: error: missing terminating ' character
                                  1'000'000'000,
                                           ^~~~~
/usr/local/include/helics/common/timeRepresentation.hpp:140:44: error: missing terminating ' character
                                  10'000'000'000,
                                            ^~~~~
/usr/local/include/helics/common/timeRepresentation.hpp:141:45: error: missing terminating ' character
                                  100'000'000'000,
                                             ^~~~~
/usr/local/include/helics/common/timeRepresentation.hpp:145:51: error: missing terminating ' character
                                  1'000'000'000'000'000};
                                                   ^~~~~~
/usr/local/include/helics/common/timeRepresentation.hpp:152:31: error: missing terminating ' character
                             10'000.0,
                               ^~~~~~~
/usr/local/include/helics/common/timeRepresentation.hpp:153:32: error: missing terminating ' character
                             100'000.0,
                                ^~~~~~~
/usr/local/include/helics/common/timeRepresentation.hpp:157:38: error: missing terminating ' character
                             1'000'000'000.0,
                                      ^~~~~~~
/usr/local/include/helics/common/timeRepresentation.hpp:158:39: error: missing terminating ' character
                             10'000'000'000.0,
                                       ^~~~~~~
/usr/local/include/helics/common/timeRepresentation.hpp:159:40: error: missing terminating ' character
                             100'000'000'000.0,
                                        ^~~~~~~
/usr/local/include/helics/common/timeRepresentation.hpp:163:46: error: missing terminating ' character
                             1'000'000'000'000'000.0};
                                              ^~~~~~~~
/usr/local/include/helics/common/timeRepresentation.hpp:135:34: error: expected '}' before numeric constant
                                  100'000,
                                  ^~~
/usr/local/include/helics/common/timeRepresentation.hpp:135:34: error: expected ',' or ';' before numeric constant
configure:11178: $? = 1
```

--- a/setup/README.md
+++ b/setup/README.md
@@ -20,7 +20,7 @@ Installing GridLAB-D with HELICS on Linux and MacOS:
 -   Type `autoreconf -if`
 -   Then run the following
     ```
-    ./configure --prefix=/path/to/gridlabd/install --with-helics=/path/to/helics/install --enable-silent-rules "CFLAGS=-g -O0 -w" "CXXFLAGS=-g -O0 -w -std=c++11" "LDFLAGS=-g -O0 -w"
+    ./configure --prefix=/path/to/gridlabd/install --with-helics=/path/to/helics/install --enable-silent-rules "CFLAGS=-g -O0 -w" "CXXFLAGS=-g -O0 -w -std=c++14" "LDFLAGS=-g -O0 -w"
     ```
 -   For Sierra users it is important that you compile HELICS and
     GridLAB-D and there prereqs with gcc and not clang. There is an

